BLSQ/iaso
# POLIO-1443: Remove test campaigns from the Campaigns CSV extract

Explain what problem this PR is resolving
- When there is no filter, it generates csv with test campaigns
- This PR is about to fix that

Related JIRA tickets : [POLIO-1443](https://bluesquare.atlassian.net/browse/POLIO-1443)

## Self proofreading checklist


## Changes
- Rename the campaign type 'Test campaigns' into 'Test campaign/on hold'
- Add filterLaunched param to confirm when the button filter has been pressed
- Make showTest to be false or true depending on the value of filterLaunched

## How to test
- Go on campaigns page 
- Download csv without any filter(there should not be test campaigns)
- Download csv with filter campaign_type at all(there should be test campaigns)

## Print screen / video
![Screenshot from 2024-02-22 16-54-45](https://github.com/BLSQ/iaso/assets/19631540/5f202d5b-da51-418a-bcf9-947bc62c2f6c)

[POLIO-1443]: https://bluesquare.atlassian.net/browse/POLIO-1443?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
